Revit_oM/Generic/FamilyLibrary.cs

```diff
@@ -22,15 +22,18 @@
 
 using System.Collections.Generic;
 using BH.oM.Base;
+using System.ComponentModel;
 
 namespace BH.oM.Adapters.Revit.Generic
 {
+    [Description("Library of Revit families that can be loaded to the model. Limited functionality at the moment.")]
     public class FamilyLibrary : BHoMObject
     {
         /***************************************************/
         /****             Public Properties             ****/
         /***************************************************/
 
+        [Description("Dictionary.")]
```

Comment:
🤣 

Like it! But would almost leave it empty given that this will all be rendered through to the api.

Reply:
Let it be... but my heart is bleeding 😆